### PR TITLE
readme: Document that just deleting the namespace might not process finalizers

### PR DESCRIPTION
**Pull request by jhrozek**
Jira: OCPBUGSM-4630

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -683,6 +683,21 @@ make run
 ```
 This is mostly useful for local development.
 
+### Note on namespace removal
+Many custom resources deployed with the compliance operators use finalizers
+to handle dependencies between objects. If the whole operator namespace gets
+deleted (e.g. with `oc delete ns/openshift-operator`), the order of deleting
```
Comment (JAORMX) [inline README.md:689]: was `ns/openshift-operator` intentional?

**Comment by jhrozek** [comment]
The tests failed because of the issue in CMP-690

File: README.md (hunk 1 above)
Comment (jhrozek) [reply README.md]: No, that was a typo thanks..

(Hello, my name is Jakub, I'm a Senior Software Engineer and I have trouble patching a README :-) ...)

**Pull request merged**